packages/stats/src/types.ts

```diff
@@ -0,0 +1,17 @@
+// common types
+
+import {ToolShape} from '@tracespace/parser'
+
+export interface DrillUsage {
+  toolShape: ToolShape,
```

Comment:
Is having the full tool shape helpful? This type might be kind of hard to use in production, because `Parser.ToolShape` includes all sorts of non-circular tools.

Do you think `diameter: number` would be sufficient rather than `toolShape: ToolShape`?

Reply:
I thought the `ToolShape` may be more useful. If the `diameter` is sufficient I will just use that.